packages/documentation/docs/integration/flow-of-funds.md

```diff
@@ -0,0 +1,119 @@
+# Flow of funds
+
```

Comment:
Should there be an intro (or a mermaid diagram) here of a typical payment flow, ie, 
1) create incoming payment on receiving ASE
2) create quote on sending ASE
3) create outgoing payment on sending ASE

?

We don't really have a section for this explanation

Reply:
Does this still apply given that it's now an event handler doc and not a doc on the flow of funds?

Reply:
I think the current state of the Event Handlers section covers most of what's necessary - what actions the ASE should make in different situations.

Should we create a distinct section named `Communicating with another Rafiki instance` to address what Max mentioned, or would that overlap with the current OP client documentation?